components/Footer.js

```diff
@@ -155,6 +155,10 @@ const messages = defineMessages({
     id: 'company.securityPolicy',
     defaultMessage: 'Security Policy',
   },
+  'company.contactUs': {
+    id: 'company.contactUs',
```

Comment:
We already have a string for this one. This will also resolve the CI issue.

```suggestion
    id: 'contactUs',
```